### ISSUE TITLE
cuda_driver.cc :1037] failed to synchronize the stop event: CUDA_ERROR_LAUNCH_FAILED

### ISSUE THREAD

**Issue by TonyLOV** [dennybritz/cnn-text-classification-tf#90]
met this problem always.Then I change a computer to run,and the same problem has occurred.

**Comment by sfailsthy** [comment]
yes,I also met this problem always~~~~(>_<)~~~~

**Comment by sfailsthy** [comment]
You can try to change some parameters.It should succeed.I tried ,and succeeded.Mine is:
# Model Hyperparameters
tf.flags.DEFINE_integer("embedding_dim", 50, "Dimensionality of character embedding (default: 128)")
tf.flags.DEFINE_string("filter_sizes", "3,4,5", "Comma-separated filter sizes (default: '3,4,5')")
tf.flags.DEFINE_integer("num_filters", 32, "Number of filters per filter size (default: 128)")
tf.flags.DEFINE_float("dropout_keep_prob", 0.5, "Dropout keep probability (default: 0.5)")
tf.flags.DEFINE_float("l2_reg_lambda", 0.0, "L2 regularization lambda (default: 0.0)")

# Training parameters
tf.flags.DEFINE_integer("batch_size", 37, "Batch Size (default: 64)")
tf.flags.DEFINE_integer("num_epochs", 200, "Number of training epochs (default: 200)")
tf.flags.DEFINE_integer("evaluate_every", 200, "Evaluate model on dev set after this many steps (default: 100)")
tf.flags.DEFINE_integer("checkpoint_every", 100, "Save model after this many steps (default: 100)")
tf.flags.DEFINE_integer("num_checkpoints", 5, "Number of checkpoints to store (default: 5)")
Other  parameters maybe perfect!
